src/index.mjs

```diff
@@ -243,16 +248,29 @@ export function getMinMax(range, margin = 0) {
   };
 }
 
+/**
+ * Scale the given points into the range [from, to].
+ * 
+ * Points closer to min will be scaled closer to 'to'.
+ * Points closer to max will be scaled closer to 'from'.
+ * 
+ * Example - scalePoints(5, 30, 8, 20, [8, 9, 10, 11, 15, 16, 20])
+ * = [30, 27.916666666666668, 25.833333333333332, 23.75, 15.416666666666666, 13.333333333333332, 5]
+ * 
+ * Note: It is not necesarry for input range to include
```

Comment:
Would you mind elaborating more on this note? I don't understand what it means.

Reply:
I meant that the range can be anything between `[min, max]`.

Suppose the `min = 0` and `max = 10`, both the following are valid range
- `range = [5, 6, 7]`
- `range = [1, 5, 6, 7, 10]`

The min doesn't have to be the minimum element present in the range. Same goes for max. While reading the code at first I thought that was the case.